core/new-gui/src/theme.less

```diff
@@ -1,999 +1,999 @@
-@alert-with-description-icon-size: 24px;
-@alert-with-description-icon-top: @alert-with-description-padding-vertical;
-
-// List
-// ---
-@list-header-background: transparent;
-@list-footer-background: transparent;
-@list-empty-text-padding: @padding-md;
-@list-item-padding: @padding-sm 0;
-@list-item-padding-sm: @padding-xs @padding-md;
-@list-item-padding-lg: 16px 24px;
-@list-item-meta-margin-bottom: @padding-md;
-@list-item-meta-avatar-margin-right: @padding-md;
-@list-item-meta-title-margin-bottom: @padding-sm;
-@list-customize-card-bg: @component-background;
-@list-item-meta-description-font-size: @font-size-base;
-
-// Statistic
-// ---
-@statistic-title-font-size: @font-size-base;
-@statistic-content-font-size: 24px;
-@statistic-unit-font-size: 16px;
-@statistic-font-family: @font-family;
-
-// Drawer
-// ---
-@drawer-header-padding: @padding-md @padding-lg;
-@drawer-body-padding: @padding-lg;
-@drawer-bg: @component-background;
-@drawer-footer-padding-vertical: @modal-footer-padding-vertical;
-@drawer-footer-padding-horizontal: @modal-footer-padding-horizontal;
-@drawer-header-close-size: 56px;
-
-// Timeline
-// ---
-@timeline-width: 2px;
-@timeline-color: @border-color-split;
-@timeline-dot-border-width: 2px;
-@timeline-dot-color: @primary-color;
-@timeline-dot-bg: @component-background;
-@timeline-item-padding-bottom: 20px;
-
-// Typography
-// ---
-@typography-title-font-weight: 600;
-@typography-title-margin-top: 1.2em;
-@typography-title-margin-bottom: 0.5em;
-
-// Upload
-// ---
-@upload-actions-color: @text-color-secondary;
-
-// Steps
-// ---
-@process-tail-color: @border-color-split;
-@steps-nav-arrow-color: fade(@black, 25%);
-@steps-background: @component-background;
-@steps-icon-size: 32px;
-@steps-icon-custom-size: @steps-icon-size;
-@steps-icon-custom-top: 0px;
-@steps-icon-custom-font-size: 24px;
-@steps-icon-top: -1px;
-@steps-icon-font-size: @font-size-lg;
-@steps-icon-margin: 0 8px 0 0;
-@steps-title-line-height: @height-base;
-@steps-small-icon-size: 24px;
-@steps-small-icon-margin: 0 8px 0 0;
-@steps-dot-size: 8px;
-@steps-dot-top: 2px;
-@steps-current-dot-size: 10px;
-@steps-desciption-max-width: 140px;
-@steps-nav-content-max-width: auto;
-@steps-vertical-icon-width: 16px;
-@steps-vertical-tail-width: 16px;
-@steps-vertical-tail-width-sm: 12px;
-
-// Notification
-// ---
-@notification-bg: @component-background;
-@notification-padding-vertical: 16px;
-@notification-padding-horizontal: 24px;
-
-//  Result
-// ---
-@result-title-font-size: 24px;
-@result-subtitle-font-size: @font-size-base;
-@result-icon-font-size: 72px;
-@result-extra-margin: 24px 0 0 0;
-
-// Image
-// ---
-@image-size-base: 48px;
-@image-font-size-base: 24px;
-@image-bg: #f5f5f5;
-@image-color: #fff;
-@image-preview-operation-size: 18px;
-@image-preview-operation-color: @text-color-dark;
-@image-preview-operation-disabled-color: fade(@image-preview-operation-color, 45%);
+// @import "ng-zorro-antd/ng-zorro-antd.less";
```

Comment:
why do we comment out this file?

Reply:
The upgrade causes some import errors of this file. 
I'll delete this file, this file was just a copy of the official default theme, deleting it doesn't make any differences.
https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less

Reply:
yeah lets delete it

Reply:
done